app/build.gradle

```diff
@@ -42,7 +42,7 @@ dependencies {
     compile 'com.android.support:recyclerview-v7:23.1.1'
     apt 'org.parceler:parceler:1.0.4'
     compile 'org.parceler:parceler-api:1.0.4'
-    compile 'de.hdodenhof:circleimageview:2.0.0'
+    compile 'com.makeramen:roundedimageview:2.2.1'
```

Comment:
needed the rounded version of this library in order to round the corners on the pet icon in the swipe card.